build/panic_reporter.go

```diff
@@ -157,11 +156,7 @@ func writeJournalTail(tailLen int, repoPath, file string) {
 			}
 			break
 		}
-		if _, err := f.Write(line); err != nil {
-			panicLog.Error(err.Error())
-			break
-		}
-		if _, err := f.Write([]byte("\n")); err != nil {
+		if _, err := f.Write(append(line, "\n"...)); err != nil {
```

Comment:
backscanner docs say that the slice returned from LineBytes shouldn't be modified -> https://pkg.go.dev/github.com/icza/backscanner?utm_source=godoc#Scanner.LineBytes

Reply:
Good catch.